main.go

```diff
@@ -0,0 +1,157 @@
+/*
+modemd - Communicates with USB modems
+Copyright (C) 2019, The Cacophony Project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package main
+
+import (
+	"log"
+	"time"
+
+	arg "github.com/alexflint/go-arg"
+	"periph.io/x/periph/host"
+)
+
+func main() {
+	err := runMain()
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+type Args struct {
+	ConfigFile   string `arg:"-c,--config" help:"path to configuration file"`
+	Timestamps   bool   `arg:"-t,--timestamps" help:"include timestamps in log output"`
+	RestartModem bool   `arg:"-r,--restart" help:"cycle the power to the USB port"`
+}
+
+func (Args) Version() string {
+	return version
+}
+
+func procArgs() Args {
+	args := Args{
+		ConfigFile: "/etc/cacophony/modemd.yaml",
+	}
+	arg.MustParse(&args)
+	return args
+}
+
+var (
+	version = "<not set>"
+)
```

Comment:
when there's just one variable there's not much benefit to defining it over multiple lines :)